src/Session.js

```diff
@@ -1,18 +1,23 @@
 import EventEmitter from 'eventemitter3'
-import { ethers } from 'ethers'
+import { Wallet } from '@ethersproject/wallet'
+import { Web3Provider } from '@ethersproject/providers'
 
 export default class Session extends EventEmitter {
-    constructor(client, options) {
+    constructor(client, options = {}) {
```

Comment:
Sorry about these changes  :D  but they were kind of necessary to use the "new" window.ethereum, or maybe it was just the signing refactor. I would've wanted to do more, actually, but maybe I will get to it later.

Reply:
and by signing I mean signing transactions generated by the Data Union endpoints. I *know* I should have left all other parts alone, such as message signing (maybe I didn't commit those changes after all?) but it was done using the same credentials and that code that directly accessed the provider was there copy-pasted. In the end it seems I didn't replace that code but the "cleanups" went in there nevertheless. Hope it wasn't too much trouble.

Reply:
Not a major issue. Just took a bit to figure out what changed and what didn't.